src/games/strategy/triplea/ai/AdvancedUtils.java

```diff
@@ -36,24 +36,6 @@ public static int getIndexOfLastUnitMatching(final List<Unit> units, final Match
     return -1;
   }
 
-  public static Unit getFirstUnitMatching(final List<Unit> units, final Match<Unit> match, final int startIndex) {
```

Comment:
@ron-murhammer: Please let us know if you think you might use any of these public static utility functions.  They exist purely as nice utilities for AI makers to use.
(Personally, I don't see as big a reason to remove them, but w/e.)